### PR TITLE
Create spectator-contest.md

### PR DESCRIPTION
**Pull request by robertdijk**

### CODE REVIEW

File: content/news/spectator-contest.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Livestream URLs"
```
Comment (mpsijm) [inline content/news/spectator-contest.md:2]:
```suggestion
title: "Spectator contest"
```

File: content/news/spectator-contest.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+date: 2022-11-24T09:00:00+01:00
+draft: false
+---
+
+There will be a spectator contest hosted on [spectator.nwerc.eu](http://spectator.nwerc.eu).
```
Comment (mpsijm) [inline content/news/spectator-contest.md:7]:
```suggestion
There will be a spectator contest hosted on [spectator.nwerc.eu](https://spectator.nwerc.eu).
```

File: content/news/spectator-contest.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+Here you can self-register your team!
+The contest will start at 10:30 on Sunday (November 27 2022). This is half an our later then the real contest.
```
Comment (mpsijm) [inline content/news/spectator-contest.md:9]:
```suggestion
The contest will start at 10:30 CET on Sunday (November 27th, 2022). This is half an our later than the start of the real contest.
```

**Review by mpsijm — CHANGES_REQUESTED** [review]

**Review by verwoerd — APPROVED** [review]

**Review by mpsijm — APPROVED** [review]
Looking good! Any reason why this PR is still marked as draft? Else it can be merged :slightly_smiling_face:

**Comment by robertdijk** [comment]
Yea I'm not yet sure if the config in domjudge is fully correct and ready. @mpsijm could you check it?

**Pull request merged**